Polar fixes

igorsaux opened the pull request:
тут будут мои legendary фиксы полярки короче.....................

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.
- [x] Я ознакомился c [Guide to Contribute](https://github.com/ChaoticOnyx/OnyxBay/blob/dev/docs/contributing.md).

Zert0X commented:
![изображение](https://user-images.githubusercontent.com/42303535/166725977-2dfc3713-852e-49ab-8ec3-b3adfb8bdd14.png)

igorsaux commented:
снегу бы ещё леер подвигать, а то он над окнами и гирдердерами, а должен быть сугубо над проводами/трубами

Commit:
fucking fixes


fill zone gap


fix stripes runtime


fix stacking unit console


fix duplicates


add lights in cargo


fix shuttles


remove unused shuttles


fix snow layer


fix emergency shuttle transit area


fix merchant shuttle


remove mining shuttle


remove pods transitions


fix elite syndicate op shuttle name


tgm


fix conflicts

TobyThorne reviewed (approved):
![image](https://user-images.githubusercontent.com/45202681/167484129-887a5423-31b3-4969-b748-969f4c1cd463.png)
вишня
